roles/nougat/tasks/main.yml

```diff
@@ -0,0 +1,24 @@
+---
+- include: dependencies.yml
+
+- name: Fetch NouGAT from GitHub
+  git: repo="{{ nougat_repo }}"
+       dest="{{ nougat_dest }}"
+       version="{{ nougat_version }}"
+       force=yes
+
+- name: Install NouGAT's requirements
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install -r {{ nougat_dest }}/requirements.txt"
+
+- name: Copy configuration
+  template: src="NouGAT_config.conf.j2" dest="{{ ngi_pipeline_conf }}/NouGAT_config.conf"
+
+- name: Install NouGAT
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install -e ."
```

Comment:
Yea, it's necessary for the entry_points mentioned here https://github.com/SciLifeLab/NouGAT/blob/master/setup.py